recipes/ai2thor/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ai2thor" %}
+{% set version = "2.4.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 336f320d3977a59db40f9409ee7de1ab8162fa2efbbfdbe714dbe14c3797195d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - aws-requests-auth
+    - botocore
+    - flask
+    - msgpack-python
+    - numpy
+    - opencv
+    - pillow
+    - progressbar2
+    - python
+    - pyyaml
+    - requests
+    - werkzeug >=0.15.0
+
+test:
+  imports:
+    - ai2thor
+    - ai2thor.util
+  requires:
+    - pip
+
+about:
+  home: https://github.com/allenai/ai2thor
+  summary: AI2 Thor is a lightweight AI framework that interacts with the Unity3d Game Engine.
+  license: Apache 2.0
+  license_file: LICENSE
```

Comment:
Since you manually packaged the license, please add a comment above the `license_file` line that includes a link to the following PR:

https://github.com/allenai/ai2thor/pull/407

This will be a visual reminder to check to see if the license file is included in the source distribution in future releases.